addons/website_mail/static/src/js/follow.js

```diff
@@ -111,14 +125,19 @@ publicWidget.registry.follow = publicWidget.Widget.extend({
         $jsFollow.removeClass('o_has_error').find('.form-control, .custom-select').removeClass('is-invalid');
 
         var email = $email.length ? $email.val() : false;
-        if (email || this.isUser) {
+        if ((email || this.isUser) && this._recaptchaLoaded) {
+            const tokenCaptcha = await this._recaptcha.getToken("website_mail");
+            if (!tokenCaptcha.token) {
+                return;
+            }
```

Comment:
Should we fail silently?
[Here](https://github.com/odoo/odoo/blob/57793ff912ac5aab8f3d3e014992965d06290bf0/addons/website_mass_mailing/static/src/js/website_mass_mailing.js#L99-L108), it displays a notification.

I tested this locally, if no recaptcha key is configured then clicking on the button does nothing.
Do we want to fail in this case too?

Reply:
Indeed failing silently does not make sense, I just fixed that.